src/Auth/Views/Account/LoggedOut.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@using IdentityServer4.Extensions
+@using Codidact.Application.Common.Interfaces
+@inject ISettingsService settings
+@model LoggedOutViewModel
+
+@{
+    // set this so the layout rendering sees an anonymous user
+    ViewData["signed-out"] = true;
+}
+
+<div class="page-header logged-out">
+    <h1>
+        Logout
+        <small>You are now logged out</small>
+    </h1>
+
+    @if (Model.PostLogoutRedirectUri != null)
+    {
+        <div>
+            Click <a class="PostLogoutRedirectUri" href="@Model.PostLogoutRedirectUri">here</a> to return to the
+            <span>@settings.GetPlatformName()</span> application.
+        </div>
+    }
+</div>
+
+@section scripts
+{
+    @if (Model.AutomaticRedirectAfterSignOut)
+    {
+        <script src="~/js/signout-redirect.js"></script>
+    }
+}
```

Comment:
I tried removing this but I had to do extensive testing and discovered the automatic sign out didn't work because callback sign out URL never came in. 
In the end I managed to remove this but I really hope that this is not some special requirement for some special situation that IdentityServer opts in for. It doesn't seem so at least.
https://github.com/misha130/core/commit/beda0a29086e2018e450f49a57265615330202ca